packages/make-styles/src/ax.ts

```diff
@@ -116,11 +116,11 @@ export function ax(dir: 'ltr' | 'rtl', classNames: (string | false | undefined)[
     const resultDefinition = resultDefinitions[property];
 
     if (isRtl) {
-      const rtlPrefix = isRtl && resultDefinition[2] ? RTL_PREFIX : '';
+      const rtlPrefix = isRtl && resultDefinition[3] ? RTL_PREFIX : '';
```

Comment:
Is this the time to introduce constants?

Reply:
btw, is there a way to force-inline a constant?

Reply:
> btw, is there a way to force-inline a constant?

Terser will do it for us, I created a sandbox to show it: https://codesandbox.io/s/terser-forked-txjd2?file=/src/index.js

> Is this the time to introduce constants?

Great idea, I will add them in a follow up PR, in this case we will be sure that it does not affect bundle size 👍